extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
How is the scrolling on iOS? I suspect we want `-webkit-overflow-scrolling: touch;` here